roles/technical_principal.md

```diff
@@ -0,0 +1,154 @@
+# Technical Principal
+
+Our Technical Principals are consultative engineers who are commercially responsible for growing Made Tech both in terms of maintaining the quality of our teams as we scale and in helping win new business.
```

Comment:
Is something like this clearer?

Our Technical Principals are consulting engineers who are commercially responsible for growing Made Tech by marrying deep technical expertise with a highly commercial perspective. Technical Principals provide senior support to delivery teams and help win new business.

Reply:
Like it (besides the word "marrying").

Reply:
don't fight it